### PR TITLE
SCP-2334 - Marlowe Dashboard - implement new design of contract screen part 1

### PR DESCRIPTION
**Pull request by hrajchert**
This PR makes the first part in the contract screen redesign:

changes

Things to note:
* When it comes to deposits the design makes a distinction between accounts and wallet, but I'm not sure if we have this information. I think we can assume that deposits always comes from "wallet -> account", but we'll have to see when we do the transaction summary if we can always tell.
* In the choice input we don't have access to the original choice bounds, so I cant use the same hack that I used for the singleChoice and sadly the input reads "1 for Report problem" etc. 
* There is a bug with lifeCycleEvent that prevents tooltips and other halogen components from rendering
* I've changed box-shadow to drop-shadow in the step cards and now the top right corner doesn't have the "shadow artifact" 🎉

Pre-submit checklist:
- Branch
    - [X] Commit sequence broadly makes sense
    - [X] Key commits have useful messages
    - [X] Relevant tickets are mentioned in commit messages
    - [X] Formatting, materialized Nix files, PNG optimization, etc. are updated
- PR
    - [X] Self-reviewed the diff
    - [X] Useful pull request description
    - [X] Reviewer requested

Pre-merge checklist:
- [ ] Someone approved it
- [ ] Commits have useful messages
- [ ] Review clarifications made it into the code
- [ ] History is moderately tidy; or going to squash-merge

### CODE REVIEW

**Review by merivale — APPROVED** [review]
Looking good. There's an issue with the balances tab looking different (and uglier) on the current step and the past steps - because in both cases the tabs use the same list of styles. But you may already know that, and in any case that can be done in part 2 (or 3...).

File: marlowe-dashboard-client/src/Dashboard/View.purs [hunk 1]
```diff
@@ -188,14 +189,29 @@ mobileMenu menuOpen =
...
+                    else
+                      [ "cursor-default" ]
+              ]
+              [ text "Dashboard" ]
+          ]
         <> case mSelectedContractState of
             Just { nickname } ->
-              [ span_ [ text ">" ]
+              [ span [ classNames [ "font-semibold" ] ] [ text ">" ] -- FIXME: change > for an Icon
+              , tooltip "Go to dashboard" (RefId "goToDashboard") Bottom
```
[the remark this answers is not in the document]
Comment (hrajchert) [reply marlowe-dashboard-client/src/Dashboard/View.purs:214]: Yeah, actually I added to discover the whole `lifeCycle` issue, could remove it if its not needed

File: web-common/src/Material/Icons.purs [hunk 2]
```diff
@@ -39,6 +39,8 @@ data Icon
   | HelpOutline
   | History
   | Home
+  | Info
+  | Language
```
[the remark this answers is not in the document]
Comment (hrajchert) [reply web-common/src/Material/Icons.purs:43]: Yeah, will add them in the next PR with a comment

**Comment by hrajchert** [comment]
Yeah, the balances are broken for now but I wanted to merge as it is as the PR was getting bloated. Will try to make shorter PR's for the outstanding tasks

**Pull request merged**